src/main/java/ua/lviv/iot/parkingServer/natscontroller/NatsParkingCreateController.kt

```diff
@@ -22,10 +23,9 @@ class NatsParkingCreateController(
     override val parser: Parser<ParkingRequest> =
         ParkingRequest.parser()
 
-    override fun generateReplyForNatsRequest(
-        request: ParkingRequest
-    ): ParkingResponse {
-        val parking: Parking = service.addEntity(converter.protoRequestToParking(request))
-        return converter.parkingToProtoResponse(parking)
+    override fun generateReplyForNatsRequest(request: ParkingRequest): Mono<ParkingResponse> {
+        val parking: Mono<Parking> = service.addEntity(converter.protoRequestToParking(request))
+        return parking
+            .map { converter.parkingToProtoResponse(it) }
```

Comment:
```suggestion
        return service.addEntity(converter.protoRequestToParking(request))
            .map { converter.parkingToProtoResponse(it) }
```

no need to split reactor chain here, try to have one chain when it's possible

Reply:
✅